src/main/java/jenkins/plugins/office365connector/utils/TeamsMentionUtils.java

```diff
@@ -0,0 +1,33 @@
+package jenkins.plugins.office365connector.utils;
+
+import org.apache.commons.lang.StringUtils;
+import hudson.model.User;
+import hudson.tasks.Mailer;
+
+public final class TeamsMentionUtils {
+
+    private TeamsMentionUtils() {
+        // utility class
+    }
+
+    public static String mentionEmail(String email) {
+        if (StringUtils.isBlank(email)) {
+            return email;
+        }
+        return "<at>" + email + "</at>";
+    }
+
+    public static String mentionUser(User user) {
```

Comment:
name of this method does not report your real intention but rather fallback scenario - I believe this is still `mentionEmail`